Sources/PointFree/Account/UpdateProfile.swift

```diff
@@ -47,7 +52,7 @@ let updateProfileMiddleware =
 
       let updateFlash: Middleware<HeadersOpen, HeadersOpen, Prelude.Unit, Prelude.Unit>
       if data.email.rawValue.lowercased() != user.email.rawValue.lowercased() {
-        updateFlash = flash(.warning, "We’ve sent an email to \(user.email) to confirm this change.")
+        updateFlash = flash(.warning, "We've sent an email to \(user.email) to confirm this change.")
```

Comment:
something about the way we encode json for cookies breaks when we us this unicode character. we should fix it asap, but I didn't want it to hold us up. I could prob write a test for this that uses a WKWebView 🤔